codeforboston/vaccinatema
# Undo revert of #33 and downgrade Next.

When I tried deploying #33, I saw 502 errors. The error in the logs was: `Error: Cannot find module 'sharp'`. It looks like this is a [known issue](https://github.com/vercel/next.js/issues/20456) with Next and that downgrading to v10.0.3 resolves it.

This PR reverts the revert of #33 and downgrades v10.0.3.

### Test Plan:
* Ensure the whole website functions as usual (with no console errors).
* Double check that the search functionality works as expected.
* Go to localhost:3002/dev and see "Hello, world!".
